junebug/channel.py

```diff
@@ -29,6 +30,13 @@ class InvalidChannelType(JunebugError):
     'xmpp': 'vumi.transports.xmpp.XMPPTransport',
 }
 
+allowed_message_fields = [
+    'transport_name', 'timestamp', 'in_reply_to', 'to_addr', 'from_addr',
+    'content', 'session_event', 'helper_metadata', 'message_id']
+# excluded fields: from_addr_type, group, provider, routing_metadata,
+# to_addr_type, from_addr_type, message_version, transport_metadata,
+# message_type, transport_type
```

Comment:
Yes, I'm thinking of creating a function that takes a vumi message, and creates a dictionary that should be returned to the user, because some of the conversion can be a bit tricky, so I think it's best to do it manually.